packages/client/src/types/types.ts

```diff
@@ -0,0 +1,2 @@
+export type ButtonEnum = 
+| "submit" | "clear" | "button"
```

Comment:
```suggestion
| "submit" | "reset" | "button"
```
Поэтому ругается TypeScript

Reply:
Спасибо! :)
Поправил